chapter_preliminaries/probability.md

```diff
@@ -82,6 +82,14 @@ import tensorflow_probability as tfp
 import numpy as np
 ```
 
+```{.python .input}
+#@tab paddle
+%matplotlib inline
+import random
```

Comment:
This should start with importing paddle as d2l since that is called below for plotting with `d2l.plt`.

```suggestion
from d2l import tensorflow as d2l
import random
```